### PR TITLE
Dashboard change to Network page to support new NetworkManager

### PR DESCRIPTION
**Pull request by BaronGreenback**
Dashboard change to reflect the network changes made in https://github.com/jellyfin/jellyfin/pull/4124.

Configuration saved to named configuration file.

Additional configuration options created.

### CODE REVIEW

File: src/controllers/dashboard/networking.js [hunk 1]
```diff
@@ -47,8 +59,18 @@ import 'emby-select';
                     config.EnableRemoteAccess = form.querySelector('#chkRemoteAccess').checked;
                     config.CertificatePath = form.querySelector('#txtCertificatePath').value || null;
                     config.CertificatePassword = form.querySelector('#txtCertPassword').value || null;
-                    config.LocalNetworkAddresses = localAddress ? [localAddress] : [];
-                    ApiClient.updateServerConfiguration(config).then(Dashboard.processServerConfigurationUpdateResult, Dashboard.processErrorResponse);
+
```
Comment (h1dden-da3m0n) [inline src/controllers/dashboard/networking.js:62]:
```suggestion
```

**Comment by PrplHaz4** [comment]
this has translation merge conflicts now

**Comment by BaronGreenback** [comment]
> this has translation merge conflicts now

thx - hadn't seen.

File: src/strings/en-gb.json [hunk 2]
```diff
@@ -1417,5 +1417,29 @@
     "ButtonUseQuickConnect": "Use Quick Connect",
     "ButtonActivate": "Activate",
     "Authorize": "Authorise",
-    "EnableAutoCast": "Set as Default"
+    "EnableAutoCast": "Set as Default",
```
Comment (MrTimscampi) [inline src/strings/en-gb.json:1420]: The key seems very different from its value here. It's usually better to make it somewhat similar, to help translators.

**Review by MrTimscampi — APPROVED** [review]

**Review by ferferga — APPROVED** [review]

**Review by joshuaboniface — APPROVED** [review]

**Pull request merged**